src/styles/header.scss

```
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-inline: 50px;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  &__logo-link {
+    display: inline-block;
+    width: 40px;
+    height: 40px;
+  }
+
+  .nav {
+    &-list {
+      display: flex;
+      list-style: none;
+    }
+
+    &-item {
+      margin-left: 20px;
+    }
+
+    &-item:first-child {
+      margin-left: 0;
+    }
+
+    &-link {
+      display: inline-block;
+      text-decoration: none;
+      color: $link-color;
+      height: $header-height;
+      line-height: $header-height;
+    }
+
+    &-link:hover {
+      color: $main-color;
+      transition-duration: $standart-duration;
+    }
```

```suggestion
    &-item {
      margin-left: 20px;
        &:first-child {
        margin-left: 0;
      }
    }

    &-link {
      display: inline-block;
      text-decoration: none;
      color: $link-color;
      height: $header-height;
      line-height: $header-height;
      &:hover {
      color: $main-color;
      transition-duration: $standart-duration;
      }
    }
```
make use of sass bem nesting
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

---

@lerastarynets oh, thx!!!